jetson/science_bridge/README.md

```diff
@@ -0,0 +1,97 @@
+Code for the science bridge program that will interpret UART messages from a Nucleo
+======================================================================================
+### About
+Writes, reads and parses NMEA like messages from the onboard 
+science nucleo to operate the science boxes and get relevant data.
+
+### Hardware
+- NVIDIA JetsonNX
+- STM32G050C8 Nucleo board
+- RX/TX connection cables 
+
+### Spectral
+Reads the 6 channel data from 3 different spectral sensors in one NMEA style sentence from an STM32 Nucleo Board over UART. 
+#### LCM Channels Publishing/Subscribed To
+**Spectral Data [publisher]** \
+Messages: [SpectralData.lcm](https://github.com/tabiosg/mrover-workspace/blob/new_science_bridge/rover_msgs/SpectralData.lcm) "/spectral_data" \
+Publishers: jetson/science_bridge \
+Subscribers: base_station/gui
+### UART Message
+`$SPECTRAL, d0_msb_ch0, d0_lsb_ch0, d0_msb_ch1, d0_lsb_ch1, d0_msb_ch2, d0_lsb_ch2, d0_msb_ch3, d0_lsb_ch3, d0_msb_ch4, d0_lsb_ch4, d0_msb_ch5, d0_lsb_ch5, d1_msb_ch0, d1_lsb_ch0, d1_msb_ch1, d1_lsb_ch1, d1_msb_ch2, d1_lsb_ch2, d1_msb_ch3, d1_lsb_ch3, d1_msb_ch4, d1_lsb_ch4, d1_msb_ch5, d1_lsb_ch5,  d2_msb_ch0, d2_lsb_ch0, d2_msb_ch1, d2_lsb_ch1, d2_msb_ch2, d2_lsb_ch2, d2_msb_ch3, d2_lsb_ch3, d2_msb_ch4, d2_lsb_ch4, d2_msb_ch5, d2_lsb_ch5,`
+
+### Triad
+Reads the 18 channel data from a spectral triad sensor in one NMEA style sentence from an STM32 Nucleo Board over UART. 
+#### LCM Channels Publishing/Subscribed To
+**Spectral Triad Data [publisher]** \
+Messages: [SpectralData.lcm](https://github.com/tabiosg/mrover-workspace/blob/new_science_bridge/rover_msgs/SpectralData.lcm) "/spectral_triad_data" \
+Publishers: jetson/science_bridge \
+Subscribers: base_station/gui
+### UART Message
+`$TRIAD, d0_msb_ch0, d0_lsb_ch0, d0_msb_ch1, d0_lsb_ch1, d0_msb_ch2, d0_lsb_ch2, d0_msb_ch3, d0_lsb_ch3, d0_msb_ch4, d0_lsb_ch4, d0_msb_ch5, d0_lsb_ch5, d1_msb_ch0, d1_lsb_ch0, d1_msb_ch1, d1_lsb_ch1, d1_msb_ch2, d1_lsb_ch2, d1_msb_ch3, d1_lsb_ch3, d1_msb_ch4, d1_lsb_ch4, d1_msb_ch5, d1_lsb_ch5,  d2_msb_ch0, d2_lsb_ch0, d2_msb_ch1, d2_lsb_ch1, d2_msb_ch2, d2_lsb_ch2, d2_msb_ch3, d2_lsb_ch3, d2_msb_ch4, d2_lsb_ch4, d2_msb_ch5, d2_lsb_ch5,`
```

Comment:
its really 1 device 18 channels but 3 chips...i guess make sure the device order matches the channels (1-18/the letters they map to) you publish to